docs/book/v4/strategies/backed-enum.md

```diff
@@ -0,0 +1,101 @@
+# BackedEnum
+
+> Available since 4.8.0
+
+> Requires PHP 8.1+
```

Comment:
```suggestion
INFO: **New Feature**
Available since version 4.8.0

MISSING: **Installation Requirements**
[Enumerations](https://www.php.net/manual/language.enumerations.overview.php) require PHP version 8.1 or higher.
```